src/test/resources/org/rust/lang/core/parser/fixtures/complete/macros.rs

```diff
@@ -145,5 +159,7 @@ fn foo() {
     asm!("nop", a = const A + 1);
     asm!("nop", in(reg) x => y, out("eax") _);
     asm!("nop", a = const 5, b = sym foo::bar, c = in(reg) _, d = out(reg) a => _);
+    sdt::asm!("nop"); // fully qualified macro call
```

Comment:
`sDt`? =D

Reply:
Oops)
Just typo. Although it doesn't matter here